Tensile/Code.py

```diff
@@ -281,6 +281,61 @@ class LocalReadInst (Inst):
   def __init__(self,*args):
     Inst.__init__(self,*args)
 
+################################################################################
+# MFMA Instruction
+################################################################################
+class  MFMAInst (Inst):
+
+  """
+  Construct a MFMA instruction from specified precision Type, aIndex, bIndex, PLR, innerUnroll:
+
+  dataType:
+  aIndex:  index value from range (0, kernel["ThreadTile0"])
+  bIndex:  index value from range (0, kernel["ThreadTile1"])
+
+  PLR:     valida values 0,1
+
+  usage Module.addCode(Code.MFMAInst())
+
+  """
+  def  __init__(self,kernel,aIdx,bIdx,PLRval,innerUnroll):
+       self.endLine = ""
+       self.version = globalParameters["CurrentISA"]
+       self.kernel  = kernel
+       self.aIdx    = aIdx
+       self.bIdx    = bIdx
+       self.PLR     = PLRval
+       self.innerUnroll = innerUnroll
```

Comment:
should this call the Inst __init__ constructor?

Reply:
will do

Reply:
done